src/webui/static/app/tvheadend.js

```diff
@@ -1386,13 +1386,17 @@ tvheadend.toLocaleFormat = function()
 	return tvh_locale_lang.replace('_','-');
 };
 
-tvheadend.toCustomDate = function(date, format) //author: meizz, improvements by pablozg
+tvheadend.toCustomDate = function(date, format) //author: meizz, improvements by pablozg and pmatis
```

Comment:
imo i'd rather see this entire comment removed, it doesn't add anything, git history tells us this :p but i'll let @Flole998 decide :)

Reply:
Yeah there's no point in doing this. If we do this everywhere the code will be a nightmare to read, so let's remove it. To me it's unclear to which part of the code it applies anyways (this line, this function, part of this function....)

Reply:
I see the point but was following the lead. I didn't feel right removing it myself nor leaving it as "blaming" someone for my massacred code changes. :) I'll remove it.